### PR TITLE
Document trigger behavior with vector/push/pop accesses.

### PR DESCRIPTION
**Pull request by timsifive**
See e-mail discussion subject:"Size for address trigger matching and vector"

### CODE REVIEW

File: Sdtrig.tex [hunk 1]
```diff
@@ -261,6 +261,15 @@ \subsection{A Extension}
...
 triggers match on AMOs is \unspecified.
 
+\subsection{Combined Accesses}
+
+Some instructions lead a hart to perform multiple memory accesses. This includes
+vector loads and stores, as well as {\tt cm.push} and {\tt cm.pop} instructions.
+The Trigger Module should match such accesses as if they all happened
+individually. E.g. a vector load should be treated as if it performed multiple
+loads of size SEW (selected element with), and {\tt cm.push} should be treated
+as if it performed multiple stores of size XLEN.
```
[the remark this answers is not in the document]
Comment (timsifive) [reply Sdtrig.tex:271]: It should be SEW for vector operations. I think this text says that. I'll split it into 2 sentences to hopefully make that more clear.

[the remark this answers is not in the document]
Comment (pdonahue-ventana) [reply Sdtrig.tex:271]: Never mind.  I was reading too fast.  This is already clear.

[the remark this answers is not in the document]
Comment (pdonahue-ventana) [reply Sdtrig.tex:271]: Now I'm slightly concerned that two sentences loses the "e.g." nature with respect to the second one.

[the remark this answers is not in the document]
Comment (timsifive) [reply Sdtrig.tex:271]: Making it a single sentence again.

**Review by pdonahue-ventana — APPROVED** [review]
Sorry for the confusion.  At least I found the typo.

**Pull request merged**